test/functional/rpc_net.py

```diff
@@ -305,7 +305,7 @@ def test_addpeeraddress(self):
         by first testing adding a tried table entry before testing adding a new table one.
         """
```

Comment:
You might also want to edit/update/remove(?) this comment. I think mentioning that we avoid collisions by adding known-working addresses to the deterministic addrman would be good.

Reply:
done.

Reply:
true, i've removed the comment in commit.